openstack/README.md

```diff
@@ -2,15 +2,17 @@
 
 ## Apply patches
 
```

Comment:
Please describe in high level what components need to be patched and what each does (at high level)

Reply:
@wmousa, I don't see you address this comment